improver_tests/nbhood/circular_kernel/test_GeneratePercentilesFromACircularNeighbourhood.py

```diff
@@ -67,49 +71,48 @@ class Test_make_percentile_cube(IrisTest):
     """Test the make_percentile_cube method from
        GeneratePercentilesFromACircularNeighbourhood."""
 
+    def setUp(self):
+        """Set up a 2D cube."""
+
+        data = np.ones((5, 5), dtype=np.float32)
+        data[2, 2] = 0
+        self.cube = set_up_variable_cube(data, spatial_grid="equalarea",)
+
     def test_basic(self):
         """Test that the plugin returns an iris.cube.Cube."""
-        cube = set_up_cube(
-            zero_point_indices=((0, 0, 2, 2),), num_time_points=1, num_grid_points=5
-        )
+
         result = GeneratePercentilesFromACircularNeighbourhood().make_percentile_cube(
-            cube
+            self.cube
         )
         self.assertIsInstance(result, Cube)
 
     def test_coord_present(self):
         """Test that the percentile coord is added."""
-        cube = set_up_cube(
-            zero_point_indices=((0, 0, 2, 2),), num_time_points=1, num_grid_points=5
-        )
+
         result = GeneratePercentilesFromACircularNeighbourhood().make_percentile_cube(
-            cube
+            self.cube
         )
         self.assertIsInstance(result.coord("percentile"), iris.coords.Coord)
         self.assertArrayEqual(result.coord("percentile").points, DEFAULT_PERCENTILES)
-        self.assertArrayEqual(result[0].data, cube.data)
-        self.assertDictEqual(cube.metadata._asdict(), result.metadata._asdict())
+        self.assertArrayEqual(result[0].data, self.cube.data)
```

Comment:
Can I be pedantic here and ask that you create an expected data array before running the processing.  That would look like:
```
expected_data = self.cube.data.copy()
...
self.assertArrayEqual(result[0].data, expected_data)
```
That catches a case where the input cube is inadvertently modified.